yaml/pois.yaml

```diff
@@ -172,6 +172,14 @@ filters:
     min_zoom: 17
     output:
       kind: {col: tags->zoo}
+  - filter: {waterway: waterfall}
+    min_zoom: CASE WHEN to_number(tags->'height', '9999.99') < 10 THEN 15
+      WHEN to_number(tags->'height', '9999.99') >= 10 and to_number(tags->'height', '9999.99') < 50 THEN 14
```

Comment:
The `CASE WHEN ... WHEN ...` acts like an `else if`, so there's no need to make the cases mutually exclusive - this would be more readable if it were just `CASE WHEN <10 THEN ... WHEN <50 THEN ...` etc...